stdpopsim/cli.py

```diff
@@ -610,6 +740,12 @@ def write_simulation_summary(engine, model, contig, samples, seed=None):
     dry_run_text += f"{indent}Mean recombination rate: {mean_recomb_rate}\n"
     dry_run_text += f"{indent}Mean mutation rate: {mut_rate}\n"
     dry_run_text += f"{indent}Genetic map: {gmap}\n"
+    # Get the information about dfe and dfe-interval if any
+    dry_run_text += f"{indent}DFE: {dfe}\n"
+    if dfe_interval is not None:
+        dry_run_text += f"{indent}DFE in interval: {dfe_interval}\n"
+    else:
+        dry_run_text += f"{indent}DFE in interval: {str([[0,contig_len]])}\n"
```

Comment:
I think that the actual interval that's used should be passed in - i.e., we do the swapping out for None above, and None won't arrive here. Right now, so that this message is accurate, we need to replicate the behavior of `None` in two places: here, and above, which can be error-prone.